src/hooks/useTransactionNonces.ts

```diff
@@ -76,7 +80,12 @@ const useTransactionNonces = ({ transaction }: Props) => {
       setLatestReceivedNonceRuntimeApi(parseInt(latestReceivedNonceCallType.toString()));
     };
 
-    getNonceByHash();
+    const getNonceOfCurrentBlock = async () => {
```

Comment:
5) But i still need to know the nonce of the current block , otherwise i would not be able to infer if the message was processed or not on the target chain ( this is the same process as before but with a more adequate name )